diff --git a/node_modules/@types/event-calendar__core/index.d.ts b/node_modules/@types/event-calendar__core/index.d.ts
index 7e1e878..e4f2cc7 100644
--- a/node_modules/@types/event-calendar__core/index.d.ts
+++ b/node_modules/@types/event-calendar__core/index.d.ts
@@ -140,6 +140,11 @@ export namespace Calendar {
         resource: Resource;
     }
 
+    interface DayHeaderContentArg {
+        date: Date;
+        text: string;
+    }
+
     interface EventClassNamesInfo {
         event: Event;
         view: View;
@@ -341,6 +346,7 @@ export namespace Calendar {
         dayCellContent?: Content | ((arg: DayCellContentArg) => Content);
         dayCellFormat?: Intl.DateTimeFormatOptions | ((d: Date) => Content);
         dayHeaderAriaLabelFormat?: Intl.DateTimeFormatOptions | ((d: Date) => Content);
+        dayHeaderContent?: Content | ((arg: DayHeaderContentArg) => Content);
         dayHeaderFormat?: Intl.DateTimeFormatOptions | ((d: Date) => Content);
         dayMaxEvents?: boolean;
         dayPopoverFormat?: Intl.DateTimeFormatOptions | ((d: Date) => Content);
